test/models/keyword_test.rb

```diff
@@ -14,4 +14,11 @@ class KeywordTest < ActiveSupport::TestCase
     keywerd = Keyword.create(keyword: "Outdoors")
     assert_not keywerd.save, "Saved keyword without the actual keyword text"
   end
+
+
+  test "should delete a keyword" do
+    keywerd = Keyword.create(keyword: "Outdoors", weight: 1)
```

Comment:
Yeah, we need to keep code clean enough for people to take care of after we're done.